src/MIDebugPackage/MIDebugPackage.csproj

```diff
@@ -202,6 +206,19 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
   </ItemGroup>
+  <ItemDefinitionGroup Condition="'$(LocalizationEnabled)'=='true'">
+    <FilesToLocalize>
+      <Languages>$(VS)</Languages>
+      <ProjectFile>$(MSBuildThisFileFullPath)</ProjectFile>
+      <HasLceComments>true</HasLceComments>
+    </FilesToLocalize>
+  </ItemDefinitionGroup>
+  <ItemGroup Condition="'$(LocalizationEnabled)'=='true'">
+    <FilesToLocalize Include="$(OutDir)OpenFolderSchema.json">
```

Comment:
@gregg-miskelly The files look like they are put in the bin folder already. There are a couple of steps that I'm still unclear on. I will also need to update the nuget package.